libs/libxx/libcxx.defs

```diff
@@ -84,7 +84,10 @@ libcxx/src/filesystem/operations.cpp_CXXFLAGS += -Wno-shadow
 #  2676 |             const basic_string __temp (__first, __last, __alloc());
 #       |                                ^~~~~~
 
-libcxx/src/filesystem/operations.cpp_CXXFLAGS += -Wno-maybe-uninitialized
+GCCVER = $(shell $(CXX) --version | grep gcc | sed -r 's/.* ([0-9]+\.[0-9]+\.[0-9]+).*/\1/')
```

Comment:
Do we expect this to only be an issue for this version and if so can we add a comment that points at the gcc issue tracker.

Reply:
in the new libcxx, this warning will be fixed. so, we can keep it as the current state. And revert it later.